templates/style.php

```diff
@@ -1,4 +1,48 @@
+<?php
+
+// Navbar background color
+$amp_navbar_background_color = get_theme_mod( 'amp_navbar_background_color', '#0087be' );
+
+// Link Color
+$amp_link_color = $amp_navbar_background_color;
+
+// Navbar Text Color
+$amp_navbar_color = get_theme_mod( 'amp_navbar_color', '#ffffff' );
+
+// Theme color settings: Dark || Light
+$theme_color_setting = get_theme_mod( 'amp_background_color', 'light' );
+
+// Set text and border color based on $theme_color_setting
+if ( $theme_color_setting == 'light' ) {
+
+	// Convert colors to greyscale for light theme color
+	// src: http://goo.gl/2gDLsp
+	$theme_color      = '#fff';
+	$text_color       = '#535353';
+	$muted_text_color = '#9f9f9f';
+	$border_color     = '#d4d4d4';
+
+} elseif ( $theme_color_setting == 'dark' ) {
+
+	// Convert and invert colors to greyscale for dark theme color
+	// src: http://goo.gl/uVB2cO
+	$theme_color      = '#111';
+	$text_color       = '#acacac';
+	$muted_text_color = '#606060';
+	$border_color     = '#2b2b2b';
+
+} else {
+
+	// Default Calypso-based colors
+	$theme_color      = '#fff';
+	$text_color       = '#3d596d';
+	$muted_text_color = '#87A6BC';
+	$border_color     = '#c8d7e1';
+
+} ?>
```

Comment:
Same as with the featured image, we'll want to move this processing to `AMP_Post_Template`